src/Ombi/ClientApp/app/requests/remainingrequests.component.html

```diff
@@ -0,0 +1,18 @@
+﻿<div *ngIf="remaining?.hasLimit">
+    <h4 id="remainingRequests" class="text-center">
+        {{remaining.remaining}}/{{remaining.limit}} requests remaining
```

Comment:
Can we add these into the translations please? Files are in `wwwroot/translations/en.json` Same with the below